admin_manual/installation/nginx_examples.rst

```diff
@@ -114,6 +114,13 @@ your nginx installation.
       error_page 403 /core/templates/403.php;
       error_page 404 /core/templates/404.php;
   
+      # Fixes Windows WebDav client error 0x80070043 "The network name cannot be found."
+      location = / {
+          if ($request_method = OPTIONS) {
+              return 301 $scheme://$server_name/remote.php/webdav/;
```

Comment:
We talking about WebDav - just files access, not calendar, address book and etc. Even official OC desktop client use `/remote.php/webdav/`